### PR TITLE
LPS-102185 SF sort?

### PR DESCRIPTION
**Pull request by csierra**

### CODE REVIEW

File: modules/apps/portal-settings/portal-settings-web/src/main/java/com/liferay/portal/settings/web/internal/portlet/action/SavePortalSettingsFormMVCActionCommand.java [hunk 1]
```diff
@@ -110,16 +141,36 @@ protected void storeSettings(
...
+					break;
+				}
+				else if (values[i].equals(Portal.TEMP_OBFUSCATION_VALUE)) {
+					values[i] = oldValues[i];
+				}
+
+				if (!values[i].equals(oldValues[i])) {
+					modifiableSettings.setValues(name, values);
+
+					break;
```
Comment (csierra) [inline modules/apps/portal-settings/portal-settings-web/src/main/java/com/liferay/portal/settings/web/internal/portlet/action/SavePortalSettingsFormMVCActionCommand.java]: hey @stian-sigvartsen 
we need to give this loop another though as we discussed on the slack channel

File: modules/apps/portal-settings/portal-settings-authentication-token-web/src/main/resources/META-INF/resources/dynamic_include/com.liferay.portal.settings.web/token.jsp [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	<aui:fieldset id='<%= renderResponse.getNamespace() + "authenticationCookies" %>'>
+
+		<%
+		if (authenticationCookies == null) {
+			authenticationCookies = new String[0];
+		}
+
+		for (int i = 0; i < authenticationCookies.length; i++) {
+			String authenticationCookie = authenticationCookies[i];
+			authenticationCookiesIndexes[i] = i;
```
Comment (csierra) [inline modules/apps/portal-settings/portal-settings-authentication-token-web/src/main/resources/META-INF/resources/dynamic_include/com.liferay.portal.settings.web/token.jsp]: @stian-sigvartsen 

we need to rethink this too

Comment (stian-sigvartsen) [reply modules/apps/portal-settings/portal-settings-authentication-token-web/src/main/resources/META-INF/resources/dynamic_include/com.liferay.portal.settings.web/token.jsp]: We just now discussed this and concluded it's a bit weird, but ok

**Comment by stian-sigvartsen** [comment]
Please see csierra/liferay-portal/pull/558

**Pull request closed without merging**